app/routes/synonyms.$synonymId.tsx

```diff
@@ -292,7 +292,7 @@ export default function () {
               unstyled
               className="padding-105 text-center"
             >
-              Cancel
+              Go back
```

Comment:
Please keep the text `Cancel`. This was from feedback on an earlier PR that we have merged.

Reply:
this one is mystifying me. I'm assuming a change snuck in during the rebase, but when I look at the branch locally, this is what I see:
<img width="805" alt="Screenshot 2024-10-22 at 1 42 24 PM" src="https://github.com/user-attachments/assets/b32da2b8-259a-486c-b557-4c18d9c38396">
I can change the B to be capitalized in "Go back" and it reflects the change but the diff says it was "Go back" previously.
<img width="611" alt="Screenshot 2024-10-22 at 1 43 56 PM" src="https://github.com/user-attachments/assets/fc16c2f2-0047-4944-90eb-5a4f1d696e65">

Reply:
if you click  "view file" in the three dot menu you can scroll down and it also says go back:
[app/routes/synonyms.$synonymId.tsx](https://github.com/nasa-gcn/gcn.nasa.gov/blob/2ac116c459bc8a799a5cbd76cfcd553b621799d1/app/routes/synonyms.%24synonymId.tsx)

Reply:
OK, but I think that this should be `Cancel`, don't you?